BeatSinger/Directory.Build.targets

```diff
@@ -0,0 +1,65 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Yeah, it's a set of build targets I'm including in the [BeatSaberModdingTools](https://github.com/Zingabopp/BeatSaberModdingTools/releases) templates. It's nicer than using post build events when other people are working on the project because it uses `BeatSaberDir` for the output and does some other things like verifying the version in manifest.json matches `AssemblyVersion` and automatically zips releases for BeatMods